Library/ViewModels/WatchProjectViewModel.swift

```diff
@@ -21,7 +21,7 @@ public protocol WatchProjectViewModelOutputs {
   var generateSelectionFeedback: Signal<(), Never> { get }
 
   /// Emits when haptic feedback should be generated
-  var generateSuccessFeedback: Signal<(), Never> { get }
+  var generateNotificationSuccessFeedback: Signal<(), Never> { get }
```

Comment:
Renamed in order to be consistent with our haptic feedback convenience functions